recipes/abseil/all/conanfile.py

```diff
@@ -95,6 +95,7 @@ def generate(self):
         if is_msvc(self):
             # see https://github.com/abseil/abseil-cpp/issues/649
             tc.preprocessor_definitions["_HAS_DEPRECATED_RESULT_OF"] = 1
+            tc.cache_variables["ABSL_MSVC_STATIC_RUNTIME"] = "ON" if is_msvc_static_runtime(self) else "OFF"
```

Comment:
```suggestion
            tc.cache_variables["ABSL_MSVC_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
```
Can be made simpler - True/False values are handled correctly by CMake.

Reply:
Nice!